.github/workflows/update-gh-pages.yml

```
@@ -1,5 +1,5 @@
 
-name: Update index.html in GH Pages
+name: Update index.html in GH Pages and generate global sitemap
```

Maybe we could rename this to:

```suggestion
name: Custom GH Pages post-processing
```